components/modals/FilterModal.tsx

```diff
@@ -1,15 +1,289 @@
+                />
+                <FilterButton
+                  label="Room 3"
+                  onPress={() => setSelectedRoom(addFilter('Room 3', selectedRoom))}
+                  selected={selectedRoom.includes('Room 3')}
+                  style={styles.right}
+                />
+              </View>
+
+              <Space size={20} />
+
+              <StyledText size="lg" font="bold">
+                Session Type
+              </StyledText>
+
+              <Space size={8} />
+
+              <View style={[styles.rowGroup, { borderColor: colors.text }]}>
+                <FilterButton
+                  label="Keynote"
+                  onPress={() => setSelectedSessionType(addFilter('Keynote', selectedSessionType))}
+                  selected={selectedSessionType.includes('Keynote')}
+                  style={styles.left}
+                />
+                <FilterButton
+                  label="Codelab"
+                  onPress={() => setSelectedSessionType(addFilter('Codelab', selectedSessionType))}
+                  selected={selectedSessionType.includes('Codelab')}
+                  isCenter
+                  style={{ borderColor: colors.text }}
+                />
+                <FilterButton
+                  label="Session"
+                  onPress={() => setSelectedSessionType(addFilter('Session', selectedSessionType))}
+                  selected={selectedSessionType.includes('Session')}
+                  style={styles.right}
+                />
+              </View>
+              <Space size={12} />
+              <View style={[styles.rowGroup, { borderColor: colors.text }]}>
+                <FilterButton
+                  label="Lightining talk"
+                  onPress={() => setSelectedSessionType(addFilter('Lightining talk', selectedSessionType))}
+                  selected={selectedSessionType.includes('Lightining talk')}
+                  style={styles.left}
+                />
+                <FilterButton
+                  label="Panel Discussion"
+                  onPress={() => setSelectedSessionType(addFilter('Panel Discussion', selectedSessionType))}
+                  selected={selectedSessionType.includes('Panel Discussion')}
+                  style={[styles.withborder, { borderColor: colors.text }]}
+                />
+              </View>
+
+              <Space size={30} />
+
+              <PrimaryButton label="Filter" onPress={handleFilter} />
+            </View>
+            <Space size={20} />
+          </SafeAreaView>
+        </View>
+      </Modal>
+    </View>
+  );
 };
 
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    alignItems: 'center',
+    backgroundColor: 'rgba(0, 0, 0, 0.8)',
+  },
+  header: {
+    width: '100%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: 8,
+  },
+  modalView: {
+    width: '100%',
+    height: '90%',
```

Comment:
![Screenshot_1689338261](https://github.com/droidconKE/droidconKE2023ReactNative/assets/28436092/972acd50-1387-41c2-8731-89033359569b)
There's still no margin at the bottom of the button. Emulator Pixel_2_API_30
Setting height '95%' does the trick but I don't know how it will render on other devices.

Reply:
I changed the height to use contants. and tested with pixel 2 API 33. it should be okay now